resources/rafter/charts/controller-manager/charts/minio/values.yaml

```diff
@@ -168,6 +168,8 @@ resources:
   requests:
     memory: 256Mi
     cpu: 250m
+  limits:
+    memory: 320Mi
```

Comment:
This values are overridden by rafter chart's values, please change the value there:

https://github.com/kyma-project/kyma/blob/903a710158685984a97e6182baf17d315590791c/resources/rafter/values.yaml#L254

Reply:
Fixed